client/tests/sdk/test_dataset_sdk.py

```diff
@@ -253,16 +254,32 @@ def test_setitem_with_auto_encode(self) -> None:
             "large_str": "abc" * 1000,
             "bytes": b"abc",
             "large_bytes": b"abc" * 1000,
+            "one_type_list": [1, 2, 3],
+            "one_type_tuple": ["a", "b", "c"],
+            "mixed_types_list": [1, "2", 3.0, ["a", "b"]],
+            "mixed_types_tuple": (
+                1,
+                "2",
+                3.0,
+                ("a", "b"),
+                Sequence([1, 2.0, "3"]),
+                {"a": 1, "b": "2", "c": [1, 3]},
```

Comment:
the case `{"a": 1, "b": "2", "c": [1, "3"]},` would fail

Reply:
fixed